coolwsd.xml.in

```diff
@@ -124,6 +124,8 @@
         <disable_server_audit type="bool" desc="Disabled server audit dialog and notification. Admin will no longer see warnings in the application user interface. This doesn't affect log file." default="false">false</disable_server_audit>
     </logging>
 
+    <slideshow desc="If true, client side presentation WebGl rendering otherwise SVG presentation " default="true">true</slideshow>
```

Comment:
1. If we want to enable it by default (webgl) we have to remember this option in not present in the already created config files. That means we need to by default assume `true` if not present (to check in the C++ code).
2. Can we put that "feature flag" at the end of config where all other feature flags are?
3. Can we change name so it will explicitly say what it does without looking into description here (when reading code)? `slideshow` -> `canvasSlideshowEnabled` or similar.
4. Please add missing `type="bool"`
5. Description: `If true, WebGl presentation rendered on the client side is enabled, otherwise interactive SVG is used`